src/search/findRelevantNotes.ts

```diff
@@ -4,7 +4,7 @@ import { getSettings } from "@/settings/model";
 import { InternalTypedDocument, Orama, Result } from "@orama/orama";
 import { TFile } from "obsidian";
 
-const MIN_SIMILARITY_SCORE = 0.3;
+const MIN_SIMILARITY_SCORE = 0.4;
```

Comment:
Did you find 0.3 shows too many low similarity notes?

Right now if similarity ranges from 0.4->1, the weighted is 0.7*(0.4->1), then the lowest weighted similarity score is roughly 0.3 i.e. the highest of links only. Is this a design choice? For some notes they have so many similar notes, the linked notes are only visible when you scroll to the bottom. Again, just reporting an observation.

Reply:
That's a great observation. I turned up the score because many seemingly unrelated notes showed up in the results. A single link will give 2.4 (0.8*0.3) after the last PR. 0.3 only applies to notes that have both ongoing and incoming links.

The weights and score threshold need more fine-tuning. I think this is something that we can keep iterating.

Reply:
I just saw some very weird results. 

A high similar note (78%) is ranked lower than a low similar note (45) with outgoing links.
0.78 * 0.7 = 0.546
0.45 * 0.7 + 0.24 = 0.555

Let me think about these formula more closely.